src/main/java/pricelistapp/pricelist/controller/UserController.java

```diff
@@ -0,0 +1,30 @@
+package pricelistapp.pricelist.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import pricelistapp.pricelist.model.RegisterUserDto;
+
+import javax.validation.Valid;
+
+@Controller
+@RequestMapping(value = "/index")
+public class UserController {
+
+    @RequestMapping(value = "/index/signup", method = RequestMethod.GET)
```

Comment:
W związku z tym, że kontroler jest oznaczony adnotacją @RequestMapping(value = "/index") to URI do metody signup będzie /index/index/signup. Czy na pewno tak chcesz?